src/java/ognl/security/OgnlSecurityManager.java

```diff
@@ -0,0 +1,333 @@
+package ognl.security;
+
+import ognl.OgnlRuntime;
+
+import java.io.FileDescriptor;
+import java.lang.reflect.ReflectPermission;
+import java.net.InetAddress;
+import java.security.Permission;
+
+/**
+ * Wraps current security manager with sensitive actions e.g. exit and exec disabled if is inside OgnlRuntime
+ * @author Yasser Zamani
+ * @since 3.1.23
+ */
+public class OgnlSecurityManager extends SecurityManager {
```

Comment:
This field could be made final (IDE suggested).

Reply:
:+1: fixed